requirements.txt

```diff
@@ -1,3 +1,4 @@
 dash >=2.0.0
+pandas >=1.3.4
```

Comment:
Intentional to add `pandas` here? In general we recommend but do not require it.